x-pack/plugins/apm/public/components/shared/KueryBar/index.tsx

```diff
@@ -48,11 +54,26 @@ export function KueryBar() {
   });
   const { urlParams } = useUrlParams();
   const location = useLocation();
+  const matchedRoutes = useMatchedRoutes();
+
   const apmIndexPatternTitle = chrome.getInjected('apmIndexPatternTitle');
   const indexPatternMissing =
     !state.isLoadingIndexPattern && !state.indexPattern;
   let currentRequestCheck;
 
+  let queryExample;
+
+  if (find(matchedRoutes, { name: TRANSACTIONS })) {
+    queryExample = 'transaction.duration.us > 300000';
+  } else if (find(matchedRoutes, { name: ERRORS })) {
+    queryExample = 'http.response.status_code >= 400';
+  } else if (find(matchedRoutes, { name: METRICS })) {
+    queryExample = 'process.pid = "1234"';
+  } else {
+    queryExample =
+      'transaction.duration.us > 300000 AND http.response.status_code = 200';
```

Comment:
Would like that the `status_code` example would kept to `400` - like; 

`http.response.status_code >= 400`